translations/messages.fr.yaml

```diff
@@ -127,9 +165,10 @@ user:
       label: Quelles sont vos compétences Croix-Rouge ?
       help: Cochez toutes vos compétences
     vulnerable:
+      help: <ul><li>malade chronique</li><li>obésité morbide</li><li>syndrome grippal</li><li>immunodéprimé</li><li>personne mineure ou personne de plus de 70 ans</li><li>avis défavorable de votre unité locale ou du pole santé (local ou territorial)</li></ul>
```

Comment:
you can maybe do:
```
help: |
         - malade chronique
         - obésité morbide
         - ...
```

And use `nl2br` in order to avoid html in translations